zerver/webhooks/insping/doc.md

```diff
@@ -0,0 +1,46 @@
+Get Insping notifications in Zulip! Insping (stylized as !nsping) is a
+simple uptime and performance monitoring tool, which notifies you when
+a website or service is up and running or down.
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+{!append-stream-name.md!}
+
+{!append-topic.md!}
+
+1. Go to [Insping's website](https://www.insping.com). If you haven't created an
+   account, sign up for a free account.
+
+2. Create an organization in the
+   [manage section](https://app.insping.com/#manage)
+   if there isn't one already.
+
+3. set an URL that you want to monitor. After it's done,
+   it should look something like this:
+   ![](/static/images/integrations/insping/001.png)
```

Comment:
Replace with:
```
3. Provide the URL you would like to monitor in the **Checks / Monitoring URLs** field, like so:
    ![](static/images/integrations/insping/001.png)
```

Also, please keep in mind that if I rephrase an entire section of text when specifying what I'd like you to change a certain section to, make sure that you replace it exactly like I suggested. Thanks! :)

Reply:
Okay!